plugins/webp-uploads/helper.php

```diff
@@ -2,14 +2,11 @@
 /**
  * Helper functions used for WebP Uploads.
  *
- * @package performance-lab
+ * @package webp-uploads
+ *
  * @since 1.0.0
  */
 
-if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Exit if accessed directly.
-}
-
```

Comment:
Why is this deleted?

Reply:
I am not sure if I missed any use cases. Since this file is just declaring functions and not invoking them, do we still need this safeguard?

Reply:
Yes.

Reply:
@mukeshpanchal27 Not sure I understand. Can you please share some info here?

Reply:
Check #945 for more details.